current_fall2025/scripts/app.py

```
@@ -177,50 +232,73 @@ def display_sources(sources: List):
 def main():
     # 1. RENDER UI ELEMENTS FIRST
     st.title("Boston Public Library Archives 🏛️")
-    st.caption("Explore history through the Digital Commonwealth collection. Ask about photographs, manuscripts, maps, and more.")
+    st.caption(
+        "Explore history through the Digital Commonwealth collection. Ask about photographs, manuscripts, maps, and more."
+    )
 
     # 2. LOAD RESOURCES
-    llm, embeddings = load_llm(), load_embeddings()
-    get_db_conn()  # Initialize DB connection
+    llm, embeddings, conn = load_llm(), load_embeddings(), get_db_conn()
```

Ruff (0.14.8)

Local variable `conn` is assigned to but never used

Remove assignment to unused variable `conn`

(F841)